src/stactools/testing/test_data.py

```diff
@@ -17,14 +17,23 @@
 
 
 class TestData:
-    @staticmethod
-    def get_path(rel_path: str) -> str:
+    def __init__(self, path):
+        """Creates a test data object for a given test script.
+
+        Initialize this from, e.g., `tests/__init__.py`:
+
+        ```
+        test_data = TestData(__file__)
+        ```
```

Comment:
Hm, I would have expected CI to catch this, because we're running the tests via `coverage` [in our scripts](https://github.com/stac-utils/stactools/pull/111/files#diff-d7ad7c7666dae3e86d4d3a850f7c4df6dcde86454e9995001ea017e44a3d35d5R30). Are you getting failures locally and/or in a `docker/cibuild`?

Reply:
Ah, sorry. I'm talking about the tests in the subpackages.

Reply:
Even when the `TestData` instance is imported from the `tests` module like https://github.com/stac-utils/stactools/pull/111/files#diff-d1792aee6ee8474d65b2ea2caaf9dd18ba3d79e72cc1b11d51aca3f789d7e380R9?

To be clear I don't love this way of doing things but it was just the first thing I thought of.

Reply:
🤦 I'm not sure what I was doing there, but importing from `tests` works fine.